test_workloads.sh

```diff
@@ -26,18 +39,26 @@ run_test() {
       exit
     fi
     # run redis server
+    rm -rf ./KVs/redis/dump.rdb
     ./KVs/redis/src/redis-server --protected-mode no > ${test_outputs_folder}/${test_name_suffix}_server.txt &
 
     # wait for server to start up
-    sleep 3
+    sleep 10
```

Comment:
Is such a sleep needed? Fine for now but we might need to change it later on if we figure out what hangs there.
Nothing to be done.